server/jobs/project_command_output_handler.go

```diff
@@ -155,7 +155,7 @@ func (p *AsyncProjectCommandOutputHandler) CloseJob(jobID string, repo models.Re
 
 	// Update job status and persist to storage if configured
 	if err := p.JobStore.SetJobCompleteStatus(jobID, repo.FullName, Complete); err != nil {
-		p.logger.Err("updating jobs status to complete", err)
+		p.logger.Err("updating jobs status to complete", err.Error())
```

Comment:
why do you need to `err.Error()` here?

Reply:
It was because I noticed the error logs were malformed! Something like` "ERRORSTRING(): updating jobs .."` but it looks like it's been fixed after the new logger change so I've removed this.